### ISSUE TITLE
Vulnerability scan failure on 0.30.0

### ISSUE THREAD

**Issue by luckoseabraham** [open-policy-agent/conftest#696]
The latest binary (0.30.0) from the releases is built using go 1.17.2 which has the vulnerability https://nvd.nist.gov/vuln/detail/CVE-2022-23806 

This was updated to the 17.7 by PR https://github.com/open-policy-agent/conftest/pull/679 

But there is no releases based on this change.

Is it possible to do a new release using change PR #679

**Comment by jalseth** [comment]
Hi @luckoseabraham, I intend on cutting a new release this weekend after addressing a few of the recently opened issues and some pending PRs.

**Comment by edbighead** [comment]
@jalseth any ideas where did `0.30.0` release go 🤔 
```bash
wget --quiet -O - "https://api.github.com/repos/open-policy-agent/conftest/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | cut -c 2-
0.29.0
```

**Comment by jalseth** [comment]
We just released v0.31.0 which is built with Go v1.18.1. Thanks again for filing this issue!
